### ISSUE TITLE
[Panel] All not collapsible panels have improperly added `aria-labelledby`

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3031]
## Current behavior

All Panels that cannot be collapsed or expanded have their content inside `div id='body' aria-labelledby='heading'`. Because of this some screen readers like VoiceOver would read Panel's heading twice and users may have troubles with placing screen reader focus on Panel's content text.
Additionally a11y add-on reports this as issue with 'ARIA prohibited attributes' because `div` without `role` cannot have `aria-labelledby`.

<img width="643" height="107" alt="Image" src="https://github.com/user-attachments/assets/871e547a-c3fa-4883-ae42-0166d40239bb" />

https://github.com/user-attachments/assets/199fa5a5-3d85-4105-aeab-284e68561d4f

## Proposed solution
Because Panel heading is well exposed to assistive technologies and it's easy to reach with screen reader there is no need to add its content to another element with `aria-labelledby`. 
It's recommended to delete `aria-labelledby` from `div id='body'` to make Panel's content easier to perceive.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS team
